README.md

```diff
@@ -158,6 +158,17 @@ It accepts a second input that allows defining the hotkey that should open the d
 
 You can also provide a custom component. To help you with that, the service exposes the `getShortcuts` method.
 
+#### `removeShortcuts`
+
+Remove previously registered shortcuts.
+
+```ts
+// Remove a single shortcut
+this.hotkeys.removeShortcuts({ keys: 'meta.a' });
```

Comment:
Why an object? take a string or string[]

Reply:
You're right!